src/components/views/settings/feedback.tsx

```diff
@@ -43,6 +43,7 @@ export const Feedback = () => {
 
 
 
+Please do not remove. This info helps the technical team assist you
```

Comment:
Do we know if they want this to be translated, since it's intended for the user to read?